packages/async-rewriter2/src/async-writer-babel.ts

```diff
@@ -72,7 +72,9 @@ export default class AsyncWriter {
         ]
       ], { code: true, ast: false })?.code as string;
     } catch (e: any) {
-      e.message = e.message.replace('unknown: ', '');
+      const { message } = e;
+      delete e.message; // e.message may have been non-writable
+      e.message = message.replace('unknown: ', '');
```

Comment:
This was a result from one of the implicit dependency updates that came with regenerating the package-lock files